test/org/opensolaris/opengrok/web/DirectoryListingTest.java

```diff
@@ -129,28 +147,28 @@ public void setUp() throws Exception {
         directory = FileUtilities.createTemporaryDirectory("directory");
 
         entries = new FileEntry[3];
-        entries[0] = new FileEntry("foo.c", "foo.c", 0, 1);
-        entries[1] = new FileEntry("bar.h", "bar.h", Long.MAX_VALUE, 0);
-        entries[2] = null;
+        entries[0] = new FileEntry("foo.c", "foo.c", 0, 112, null);
+        entries[1] = new FileEntry("bar.h", "bar.h", Long.MAX_VALUE, 0, null);
+        // Will test getSimplifiedPath() behavior for ignored directories.
+        // Use DIRECTORY_INTERNAL_SIZE value for length so it is checked as the directory
+        // should contain "-" (DIRECTORY_SIZE_PLACEHOLDER) string.
+        entries[2] = new FileEntry("subdir", "subdir/", 0, DIRECTORY_INTERNAL_SIZE, Arrays.asList(
+                new FileEntry[]{
+                    new FileEntry("SCCS", "SCCS", 0, DIRECTORY_INTERNAL_SIZE, Arrays.asList(
```

Comment:
should this have the `href` argument with trailing slash given this is a directory ?